conjure-java-jackson-serialization/src/main/java/com/palantir/conjure/java/serialization/ObjectMappers.java

```diff
@@ -221,18 +223,26 @@ public static ObjectMapper withDefaultModules(ObjectMapper mapper) {
                 .disable(DeserializationFeature.ACCEPT_FLOAT_AS_INT);
     }
 
-    private static JsonFactory jsonFactory() {
-        JsonFactory jsonFactory = new InstrumentedJsonFactory();
-        // Interning introduces excessive contention https://github.com/FasterXML/jackson-core/issues/946
-        jsonFactory.disable(JsonFactory.Feature.INTERN_FIELD_NAMES);
-        return jsonFactory;
+    /** Creates a new {@link JsonFactory} configured with Conjure defaults. */
+    public static JsonFactory jsonFactory() {
+        return withDefaults(InstrumentedJsonFactory.builder()).build();
     }
 
-    private static SmileFactory smileFactory() {
-        return InstrumentedSmileFactory.builder()
-                .disable(SmileGenerator.Feature.ENCODE_BINARY_AS_7BIT)
-                // Interning introduces excessive contention https://github.com/FasterXML/jackson-core/issues/946
-                .disable(JsonFactory.Feature.INTERN_FIELD_NAMES)
+    /** Creates a new {@link SmileFactory} configured with Conjure defaults. */
+    public static SmileFactory smileFactory() {
+        return withDefaults(InstrumentedSmileFactory.builder().disable(SmileGenerator.Feature.ENCODE_BINARY_AS_7BIT))
                 .build();
     }
+
+    /** Creates a new {@link CBORFactory} configured with Conjure defaults. */
+    public static CBORFactory cborFactory() {
+        return withDefaults(CBORFactory.builder()).build();
+    }
+
+    /** Configures provided JsonFactory with Conjure default settings. */
+    private static <F extends JsonFactory, B extends TSFBuilder<F, B>> B withDefaults(B builder) {
+        return builder
+                // Interning introduces excessive contention https://github.com/FasterXML/jackson-core/issues/946
+                .disable(JsonFactory.Feature.INTERN_FIELD_NAMES);
+    }
```

Comment:
Unclear if we want this to be public like `withDefaultModules(ObjectMapper)`/etc. I opted to keep it private to push consumers toward concrete factories where we can add instrumentation.

Reply:
Based on some other usages, making this public seems beneficial to consolidate functionality